<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-us" lang="en-us">
<head>
  <link href="//gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">
  <meta name="generator" content="Hugo 0.58.3" />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>the MaD Seminar, Spring 19 &middot; Math and Data</title>

  
  <link type="text/css" rel="stylesheet" href="http://mad.cds.nyu.edu/css/print.css" media="print">
  <link type="text/css" rel="stylesheet" href="http://mad.cds.nyu.edu/css/poole.css">
  <link type="text/css" rel="stylesheet" href="http://mad.cds.nyu.edu/css/syntax.css">
  <link type="text/css" rel="stylesheet" href="http://mad.cds.nyu.edu/css/hyde.css">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Abril+Fatface|PT+Sans:400,400i,700">


  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
  <link rel="shortcut icon" href="/favicon.png">

  
  
</head>

  <body class=" ">
  <div class="sidebar">
  <div class="container sidebar-sticky">
    <div class="sidebar-about">
      <IMG SRC="http://mad.cds.nyu.edu//img/logo2.png" ALT="some text" WIDTH=400 HEIGHT=100>
      <a href="http://mad.cds.nyu.edu/"><h1>Math and Data</h1></a>
      <p class="lead">
       Center for Data Science and Courant Institute, NYU 
      </p>
    </div>

    <ul class="sidebar-nav">
      
      
      
         
                <li>
                    <a href="/about/">
                        
                        <span>About</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/people/corefaculty/">
                        
                        <span>Faculty</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/people/phds/">
                        
                        <span>PhD/MsC Students</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/people/postdocs/">
                        
                        <span>Postdocs and Fellows</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/reading/">
                        
                        <span>Reading Groups</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/seminar/">
                        
                        <span>the MAD Seminar</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/madplus/">
                        
                        <span>the MAD&#43; Seminar</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/micsem/">
                        
                        <span>the MIC Seminar</span>
                    </a>
                </li>
        
      
         
                <li>
                    <a href="/people/miscpeople/">
                        
                        <span>Visitors and Alumni</span>
                    </a>
                </li>
        
      
    </ul>

  </div>
</div>



    <main class="content container">
    <div class="post">
  <h1>the MaD Seminar, Spring 19</h1>
  

<p>The MaD seminar features leading specialists at the interface
of Applied Mathematics, Statistics and Machine Learning. It is partly supported by the Moore-Sloan Data Science Environment at NYU.</p>

<p><strong>Room:</strong> Auditorium Hall 150, Center for Data Science, NYU, <a href="https://www.google.com/maps/place/NYU+Center+for+Data+Science/@40.735016,-73.9969907,17z/data=!3m1!4b1!4m5!3m4!1s0x89c2599787834ad9:0x5dd8af15d9fbc8a3!8m2!3d40.735016!4d-73.994802">60 5th ave</a>.</p>

<p><strong>Time:</strong> 2:00pm-3:00pm, Reception will follow.</p>

<p><strong>Subscribe to the Seminar Mailing list <a href="http://cims.nyu.edu/mailman/listinfo/mad">here</a></strong></p>

<h3 id="schedule-with-confirmed-speakers">Schedule with Confirmed Speakers</h3>

<table>
<thead>
<tr>
<th>Date</th>
<th align="center">Speaker</th>
<th align="center">Title</th>
</tr>
</thead>

<tbody>
<tr>
<td>Feb 21</td>
<td align="center"><a href="https://scholar.google.com/citations?user=cn_FoswAAAAJ&amp;hl=en">Jeffrey Pennington (Google)</a></td>
<td align="center"><a href="#jeffrey">Dynamical Isometry and a Mean Field Theory of Signal Propagation in Deep Neural Networks</a></td>
</tr>

<tr>
<td>Feb 28</td>
<td align="center"><a href="https://web.stanford.edu/~elalaoui/">Ahmed El Alaoui (Stanford)</a></td>
<td align="center"><a href="#ahmed">Efficient Z_q synchronization on the square lattice</a></td>
</tr>

<tr>
<td>Mar 7</td>
<td align="center"><a href="http://krzakala.org">Florent Krzakala (ENS)</a></td>
<td align="center"><a href="#florent">Passed &amp; Spurious: Descent algorithms and local minima in a spiked matrix-tensor model</a></td>
</tr>

<tr>
<td>Mar 14</td>
<td align="center"><a href="https://www.stat.uchicago.edu/~rina/">Rina Foygel Barber (Chicago)</a></td>
<td align="center"><a href="#rina">Distribution free prediction: Is conditional inference possible?</a></td>
</tr>

<tr>
<td>Mar 21</td>
<td align="center">(Spring break)</td>
<td align="center"></td>
</tr>

<tr>
<td>Mar 28</td>
<td align="center"><a href="https://sites.google.com/site/victormpreciado/">Victor Preciado (UPenn)</a></td>
<td align="center"><a href="#victor">From Local Network Structure to Global Graph Spectrum</a></td>
</tr>

<tr>
<td>Apr 4</td>
<td align="center"></td>
<td align="center"></td>
</tr>

<tr>
<td>Apr 11</td>
<td align="center"><a href="https://www.samuelbhopkins.com/">Sam Hopkins (UC Berkeley)</a></td>
<td align="center"><a href="#sam">Mean Estimation with Sub-Gaussian Rates in Polynomial Time</a></td>
</tr>

<tr>
<td>Apr 18</td>
<td align="center"><a href="https://www.cs.princeton.edu/~arora/">Sanjeev Arora (Princeton and IAS)</a></td>
<td align="center"><a href="#arora">A theory for representation learning via contrastive objectives</a></td>
</tr>

<tr>
<td>Apr 25</td>
<td align="center"></td>
<td align="center"></td>
</tr>

<tr>
<td>May 2</td>
<td align="center"><a href="http://www.math.ucsd.edu/~rsaab/">Rayan Saab (UC San Diego)</a></td>
<td align="center"><a href="#rayan">New and Improved Binary Embeddings of Data</a></td>
</tr>

<tr>
<td>May 9</td>
<td align="center"><a href="https://satijalab.org/">Rahul Satija (NY Genome Center)</a></td>
<td align="center"><a href="#rahul">Modeling and integration of single-cell sequencing data</a></td>
</tr>
</tbody>
</table>

<hr />

<p><a href="https://mathsanddatanyu.github.io/website/seminar_fall2018/">Schedule Fall 2018</a></p>

<p><a href="https://mathsanddatanyu.github.io/website/seminar_spring2018/">Schedule Spring 2018</a></p>

<p><a href="https://mathsanddatanyu.github.io/website/seminar_fall2017/">Schedule Fall 2017</a></p>

<p><a href="https://mathsanddatanyu.github.io/website/seminar_spring2017/">Schedule Spring 2017</a></p>

<hr />

<h3 id="abstracts">Abstracts</h3>

<h4 id="a-name-rahul-a-rahul-satija-modeling-and-integration-of-single-cell-sequencing-data"><a name="rahul"></a> Rahul Satija: Modeling and integration of single-cell sequencing data</h4>

<p>New technologies for single-cell sequencing are giving rise to datasets that will define a ‘parts list’ of cell types for human beings, yet are characterized by extensive noise and sparse measurements. I will discuss two statistical learning methods for the analysis and interpretation of these data. We propose a regularization procedure to fit models of technical variation that can be applied to generalized linear models, factor analysis, or nonlinear autoencoders. In addition, we develop a procedure based on diagonalized canonical correlation analysis to identify correspondences across different experiments, enabling us to ‘align’ and compare datasets from different laboratories, technologies, and species.</p>

<h4 id="a-name-rayan-a-rayan-saab-new-and-improved-binary-embeddings-of-data"><a name="rayan"></a> Rayan Saab: New and Improved Binary Embeddings of Data</h4>

<p>We discuss two related problems that arise in the acquisition and processing of high-dimensional data. First, we consider distance-preserving fast binary embeddings. Here we propose fast methods to replace points from a subset of RN with points in a lower-dimensional cube {±1}m, which we endow with an appropriate function to approximate Euclidean distances in the original space. Second, we consider a problem in the quantization (i.e., digitization) of compressed sensing measurements. Here, we deal with measurements of approximately sparse signals as well as with measurements of manifold valued data. Our results apply not only to Gaussian measurement matrices but also to those drawn from bounded orthonormal systems and partial circulant ensembles, both of which arise naturally in applications and admit fast transforms. In all these problems we show state-of-the art error bounds, and to our knowledge, some of our results are the first of their kind.</p>

<h4 id="a-name-arora-a-sanjeev-arora-a-theory-for-representation-learning-via-contrastive-objectives"><a name="arora"></a> Sanjeev Arora: A theory for representation learning via contrastive objectives</h4>

<p>Representation learning seeks to represent complicated data (images, text etc.) using a vector embedding, which can then be used to solve complicated new classification tasks using simple methods like a linear classifier. Learning such embeddings is an important type of unsupervised learning (learning from unlabeled data) today. Several recent methods leverage pairs of “semantically similar” data points (eg sentences occuring next to each other in a text corpus). We call such methods contrastive learning (another term would be “like word2vec”) and propose a theoretical framework for analysing them. The challenge for theory here is that the training objective seems to have little to do with the downstream task. Our framework bridges this challenge and can show provable guarantees on the performance of the learnt representation on downstream classification tasks. I’ll show experiments supporting the theory.</p>

<p>The talk will be self-contained.
(Joint work with Hrishikesh Khandeparkar, Mikhail Khodak, Orestis Plevrakis, and Nikunj Saunshi.)</p>

<p>Bio: Sanjeev Arora is Charles C. Fitzmorris Professor of Computer Science at Princeton University and Visiting Professor in Mathematics at the Institute for Advanced Study. He works on theoretical computer science and theoretical machine learning. He has received the Packard Fellowship (1997), Simons Investigator Award (2012), Gödel Prize (2001 and 2010), ACM Prize in Computing (formerly the ACM-Infosys Foundation Award in the Computing Sciences) (2012), and the Fulkerson Prize in Discrete Math (2012). He is a fellow of the American Academy of Arts and Sciences and member of the National Academy of Science.</p>

<h4 id="a-name-sam-a-sam-hopkins-mean-estimation-with-sub-gaussian-rates-in-polynomial-time"><a name="sam"></a> Sam Hopkins: Mean Estimation with Sub-Gaussian Rates in Polynomial Time</h4>

<p>We study polynomial time algorithms for estimating the mean of a multivariate random vector under very mild assumptions: we assume only that the random vector X has finite mean and covariance. This allows for X to be heavy-tailed. In this setting, the radius of confidence intervals achieved by the empirical mean are exponentially larger in the case that X is Gaussian or sub-Gaussian. That is, the empirical mean is poorly concentrated.
We offer the first polynomial time algorithm to estimate the mean of X with sub-Gaussian-size confidence intervals under such mild assumptions. That is, our estimators are exponentially better-concentrated than the empirical mean. Our algorithm is based on a new semidefinite programming relaxation of a high-dimensional median. Previous estimators which assumed only existence of finitely-many moments of X either sacrifice sub-Gaussian performance or are only known to be computable via brute-force search procedures requiring time exponential in the dimension.</p>

<p>Based on <a href="https://arxiv.org/abs/1809.07425">https://arxiv.org/abs/1809.07425</a> to appear in Annals of Statistics</p>

<h4 id="a-name-victor-a-victor-preciado-from-local-network-structure-to-global-graph-spectrum"><a name="victor"></a> Victor Preciado: From Local Network Structure to Global Graph Spectrum</h4>

<p>Using methods from algebraic graph theory and convex optimization we study the relationship between local structural features of a network and global spectral properties. In particular, we derive expressions for the so-called spectral moments of  a graph in terms of local structural measurements, such as subgraph densities. Furthermore, we propose a series of semidefinite programs to compute bounds on the spectral radius, and other spectral properties, from a truncated sequence of spectral moments. Using our tools, we illustrate how important spectral properties of real-world networks are strongly constrained by local structural features.</p>

<h4 id="a-name-rina-a-rina-foygel-barber-distribution-free-prediction-is-conditional-inference-possible"><a name="rina"></a> Rina Foygel Barber: Distribution free prediction: Is conditional inference possible?</h4>

<p>We consider the problem of distribution-free predictive inference, with the goal of producing predictive coverage guarantees that hold conditionally rather than marginally. Existing methods such as conformal prediction offer marginal coverage guarantees, where predictive coverage holds on average over all possible test points, but this is not sufficient for many practical applications where we would like to know that our predictions are valid for a given individual, not merely on average over a population. On the other hand, exact conditional inference guarantees are known to be impossible without imposing assumptions on the underlying distribution. In this work we aim to explore the space in between these two, and examine what types of relaxations of the conditional coverage property would alleviate some of the practical concerns with marginal coverage guarantees while still being possible to achieve in a distribution-free setting. This work is joint with Emmanuel Candes, Aaditya Ramdas, and Ryan Tibshirani.</p>

<h4 id="a-name-florent-a-florent-krzakala-passed-spurious-descent-algorithms-and-local-minima-in-a-spiked-matrix-tensor-model"><a name="florent"></a> Florent Krzakala: Passed &amp; Spurious: Descent algorithms and local minima in a spiked matrix-tensor model</h4>

<p>Gradient-descent-based algorithms and their stochastic versions &ldquo;with temperature&rdquo; have widespread applications in machine learning and statistical inference. In this talk I will discuss the interplay between the loss landscape and the performances of these algorithms in a prototypical learning problem, the spiked matrix-tensor model, where precise quantitative computations can be performed. In particular, I will discuss the Kac-Rice characterization of the landscape, the role of the minima, and the comparaison with message-passing algorithms.</p>

<p>Based on joint works with G. Biroli, C. Chammarota, S. Sarao, P. Urbani and L. Zdeborova: arXiv:1812.09066 &amp; arXiv:1902.00139</p>

<h4 id="a-name-ahmed-a-ahmed-el-alaoui-efficient-z-q-synchronization-on-the-square-lattice"><a name="ahmed"></a> Ahmed El Alaoui: Efficient Z_q synchronization on the square lattice</h4>

<p>Group synchronization is an inverse problem on a graph: given a group g and a graph G, every vertex of G is assigned an element from g. One observes a noisy version of the group difference of the endpoints of every edge in G, and the task is to estimate the original assignment. This problem is relevant to computer vision, microscopy, and is a close relative to the problem of community detection in a network. Abbe et al. (2017) studied this problem for a variety of compact groups g when the graph G is the d-dimensional Euclidean lattice.</p>

<p>They established the existence of a phase transition in terms of the noise level separating a regime where recovery is possible from a regime where it is not. I will speak about the special case of the cyclic group g = Z/qZ on the d-dimensional lattice. I will show that under mild assumptions, recovery is efficiently possible whenever it is information-theoretically so.</p>

<p>The algorithm has a multi-scale structure and its analysis relies on a lattice-renormalization scheme. The fact that a &ldquo;possible-but-hard phase&rdquo;, where recovery is possible but computationally hard, is absent here extends far beyond group synchronization or lattices. Time permitting, I will show that any inference problem built on a graph of sub-exponential growth is computationally easy.</p>

<h4 id="a-name-jeffrey-a-jeffrey-pennington-dynamical-isometry-and-a-mean-field-theory-of-signal-propagation-in-deep-neural-networks"><a name="jeffrey"></a> Jeffrey Pennington: Dynamical Isometry and a Mean Field Theory of Signal Propagation in Deep Neural Networks</h4>

<p>In recent years, many state-of-the-art models in deep learning have utilized increasingly deep architectures, with some successful models like deep residual networks employing hundreds or even thousands of layers. In sequence modeling, recurrent neural networks are often trained over a similarly large number of time steps. Yet despite their widespread use, deep neural networks remain notoriously difficult to train. In this talk, I will develop a theory of signal propagation in deep networks and argue that training is feasible precisely when signals can propagate without attenuation or distortion. I will provide a theoretical characterization of these conditions, which amounts to an architecture-dependent initialization scheme. Using this type of initialization, I will show that it is possible to train vanilla convolutional networks with over 10,000 layers and that convergence rates for recurrent networks can be improved by orders of magnitude.</p>

</div>


    </main>

    
  </body>
</html>
